django/utils/translation/trans_null.py

```diff
@@ -13,7 +13,7 @@ def gettext(message):
 
 
 def ngettext(singular, plural, number):
-    if number == 1:
+    if number <= 1:
```

Comment:
Since negative years or negative plurals do not make sense, how about limit number to 0 and 1.